server.js

```diff
@@ -14,7 +18,63 @@ app.get("/", function (request, response) {
 });
 
 // TODO add your routes and helper functions here
+const maxId = Math.max(...bookings.map(ele => ele.id));
+// create a new booking
+app.post("/bookings", (req, res) => {
+
+  // for (const key in bookings[0]) {
+  //   if (!req.body[key] && key !== "id") {
+  //     res.status(400).send("Please enter all fields");
+  //   }
+  // }
+
+  // if (validator.validate(req.body.email) === false) {
+  //   res.status(400).send("Please enter a valid email");
+  // }
+
+  // if (moment(req.body.checkInDate) > moment(req.body.checkOutDate)) {
+  //   res.status(400).send("Please enter a valid dates");
+  // }
+  const newBooking = {
+    id: ++maxId,
+    roomId: req.body.roomId,
+    title: req.body.title,
+    firstName: req.body.firstName,
+    username: req.body.username,
+    email: req.body.email,
+    checkInDate: req.body.checkInDate,
+    checkOutDate: req.body.checkOutDate,
+  };
+
+  bookings.push(newBooking);
+  res.json(newBooking);
+})
+
+// read all bookings
+app.get("/bookings", (req, res) => {
+  if (bookings.length <= 0) {
+    res.status(500).send("No Available Data");
+  }
+  res.json(bookings);
```

Comment:
Can this line of code run if line 56 has run?

Reply:
should I use return after line 56?